app/components/badge-maintenance.js

```diff
@@ -0,0 +1,46 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+    tagName: 'span',
+    classNames: ['badge'],
+    escapedStatus: computed('badge', function() {
+        return this.get('badge.attributes.status').replace('-', '--');
+    }),
+    none: computed('badge', function() {
+        if (this.get('badge.attributes.status') === 'none'
+            || this.get('badge.attributes.status') == undefined) {
+            return true;
+        }
+        return false;
+    }),
+    status: alias('badge.attributes.status'),
+    color: computed('badge', function() {
+        let color = 'undefined';
+        switch (this.get('badge.attributes.status')) {
+        case 'actively-developed':
+            color = 'brightgreen';
+            break;
+        case 'passively-maintained':
+            color = 'yellowgreen';
+            break;
+        case 'as-is':
+            color = 'yellow';
+            break;
+        case 'experimental':
+            color = 'blue';
+            break;
+        case 'looking-for-maintainer':
+            color = 'orange';
+            break;
+        case 'deprecated':
+            color = 'red';
+            break;
+        }
+        return color;
+    }),
+    text: computed('badge', function() {
+        return 'Maintenance intention for this crate';
```

Comment:
Hm, this `text` function isn't really computing anything, right? It's not using `badge` for anything... I think this could just be `text: 'Maintenance intention for this crate',` like `tagName` and `classNames` are defined.